static/js/modules/filter-set.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/* global window */
+
+var $ = require('jquery');
+var _ = require('underscore');
+var URI = require('URIjs');
+
+var Filter = require('./filters').Filter;
+
+var KEYCODE_ENTER = 13;
+
+function FilterSet(elm) {
+  this.$elm = $(elm);
```

Comment:
Super small thing, but haven't we been going with `this.$body` for the top-most selector?